From be6a324c450c40a2c2c9461fcd8258ec07338d0a Mon Sep 17 00:00:00 2001
From: Matthias Reichl <hias@horus.com>
Date: Wed, 13 Nov 2024 00:14:13 +0100
Subject: [PATCH 1/3] Revert "DTS: bcm2712; re-enable SD slot CQE by default on
 Pi 5"

This reverts commit 48a15bc46004025776880a091d47a22e03449acc.

CQE still seems to experimental and cause issues with some cards
so keep it disabled by default for now.
---
 arch/arm/boot/dts/overlays/README                | 6 +++---
 arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts | 1 -
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index 5c6344eb6f98..af1d82cffd47 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -378,9 +378,9 @@ Params:
                                 non-lite SKU of CM4).
                                 (default "on")
 
-        sd_cqe                  Set to "off" to disable Command Queueing if you
-                                have an incompatible Class A2 SD card
-                                (Pi 5 only, default "on")
+        sd_cqe                  Use to enable Command Queueing on the SD
+                                interface for faster Class A2 card performance
+                                (Pi 5 only, default "off")
 
         sd_overclock            Clock (in MHz) to use when the MMC framework
                                 requests 50MHz
diff --git a/arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts b/arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts
index a63ee4b678a1..56d416e82dd5 100644
--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts
@@ -365,7 +365,6 @@ &sdio1 {
 	sd-uhs-sdr50;
 	sd-uhs-ddr50;
 	sd-uhs-sdr104;
-	supports-cqe;
 	cd-gpios = <&gio_aon 5 GPIO_ACTIVE_LOW>;
 	//no-1-8-v;
 	status = "okay";
-- 
2.39.5

